<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lobby</class>
 <widget class="QWidget" name="lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#lobby {
    background-color : rgba(102, 102, 102, 255);
}

QLabel {
    color: white;
}

QLineEdit {
    background-color: rgb(102,102,102);
  color: rgb(255, 255, 255);
}

QTreeWidget {
    background-color: rgb(102,102,102);
    color: white
}

QPushButton {
    border: 1px solid lightgray;
    background-color: rgb(84,33,0);
    color: white;
    padding: 3;
}

QPushButton:hover {
    border: 1px solid white;
    background-color: rgb(122,40,0);
}

QPushButton:pressed {
    border: 1px solid white;
    background-color: rgb(125,41,0);
    color: white;
}

QPushButton:disabled {
    background-color: rgb(40, 40, 40);
}

QTabBar::tab {
    border: 2px solid lightgray;
    background-color: rgb(100,41,0);
    color: lightgray;
    padding: 2;
    margin-right: 5px;
    margin-left: 5px;
  border-bottom: 0;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    color: white;
    background-color: rgb(122,40,0);
}

QTabBar::tab:selected {
    background-color: rgb(122,40,0);
    border: 2px solid white;
  border-bottom: 0;
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}

QHeaderView::section {
    border: 1px solid white;
    background-color: rgb(122,40,0);
    margin-right: 2px;
  padding: 1;
}

QHeaderView::section:pressed {
    background-color: rgb(84,33,0);
}

QHeaderView::up-arrow
{
    image: url(&quot;:/resource/ui/lobby_assets/up-arrow.png&quot;);
}

QHeaderView::down-arrow
{
    image: url(&quot;:/resource/ui/lobby_assets/down-arrow.png&quot;);
}

QScrollBar:vertical {
  background-color: rgba(140,140,140, 255);
  color : white
}

QScrollBar:horizontall {
  background-color: rgba(140,140,140, 255);
  color : white
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <widget class="QWidget">
         <attribute name="title">
          <string>Public</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="public_search">
            <property name="placeholderText">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="public_tree">
            <property name="styleSheet">
             <string notr="true">QHeaderView, QTreeView {
background-color: rgba(140, 140, 140, 255);
color: rgb(255, 255, 255);
}

QToolTip {
color: black;
}</string>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="textAlignment">
              <set>AlignLeading|AlignVCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Index</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QPushButton" name="direct_connect">
              <property name="text">
               <string>Direct Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_to_favorite">
              <property name="text">
               <string>Add To Favorites</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget">
         <attribute name="title">
          <string>Favorites</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="favorite_tree">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(140, 140, 140, 255);
color: rgb(255, 255, 255);

QToolTip {
  color: black;
};</string>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Index</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QPushButton" name="add_favorite">
              <property name="text">
               <string>Add Server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remove_favorite">
              <property name="text">
               <string>Remove Server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edit_favorite">
              <property name="text">
               <string>Edit Server</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="refresh">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="lobby_controls" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="version">
              <property name="text">
               <string>Version %1</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="options">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="about">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>About</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel">
           <property name="styleSheet">
            <string notr="true">color: #F8A818;
background-color: #383838;
border: 1px solid white;</string>
           </property>
           <property name="text">
            <string>Server Info</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="description">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(140, 140, 140, 255);
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="connect">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel">
           <property name="styleSheet">
            <string notr="true">color: #F8A818;
background-color: #383838;
border: 1px solid white;</string>
           </property>
           <property name="text">
            <string>Message of the Day</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="motd">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(140, 140, 140, 255);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
